.vscode/settings.json

```diff
@@ -12,5 +14,8 @@
     "node_modules": false
   },
   "typescript.tsdk": "node_modules/typescript/lib",
-  "debug.node.autoAttach": "on"
+  "debug.node.autoAttach": "on",
+  "yaml.schemas": {
+    "https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json": "/syntaxes/*.yaml"
```

Comment:
I found this JSONSchema which *somehow* adds a tiny little bit of documentation to the textmate format.